.ci/pytorch/test.sh

```diff
@@ -1000,8 +1000,25 @@ test_docs_test() {
 
 test_executorch() {
   # Test torchgen generated code for Executorch.
-  echo "Testing Executorch op registration"
+  echo "Testing ExecuTorch op registration"
   "$BUILD_BIN_DIR"/test_edge_op_registration
+
+  pushd /executorch
+
+  echo "Install torchvision and torchaudio"
+  # TODO(huydhn): Switch this to the pinned commits on ExecuTorch once they are
+  # there.  These libraries need to be built here, and not part of the Docker
+  # image because they require the target version of torch to be installed first
+  pip_install --no-use-pep517 --user "git+https://github.com/pytorch/audio.git"
+  pip_install --no-use-pep517 --user "git+https://github.com/pytorch/vision.git"
+
+  echo "Run ExecuTorch regression tests for some models"
+  # NB: This is a sample model, more can be added here
+  export PYTHON_EXECUTABLE=python
+  # shellcheck disable=SC1091
+  source .ci/scripts/test.sh mv3 cmake xnnpack-quantization-delegation ''
```

Comment:
This generally looks good. Has the executorch team confirmed that the tests being run are a good representative set for what they need?

Given the amount of time spent setting up the environment for this test, the incremental cost of adding a couple more tests would be very cheap

Reply:
> This generally looks good. Has the executorch team confirmed that the tests being run are a good representative set for what they need?

Yeah, I will need to confirm that part with executorch

@guangy10  Do you think this one is ok as the first iteration or we need to test more things here?

Reply:
As we have moved most non-critical and slow-running jobs off pull.yaml in ET, I think we should run most of PR jobs if not all, maybe exclude doc build and linter jobs. I think one benefit is that whenever we add/remove PR jobs in ET, it will automatically reflect to the ET jobs in PT, with reasonable delay (nightly)

Reply:
Ok, that makes sense.  I think I will break that part into a separate PR after this one.  The reason is that running the same number of ET jobs from pull.yml will requires some level of sharding here so that they can be run on more than one runner.  This part is definitely worth another PR in itself.